### PR TITLE
Better fax & headset message notice

### PR DESCRIPTION
**Pull request by Ty-Omaha**
Makes the fax notice & headset message that appears in chat to be more noticeable so people dont miss it
Wording is now _"Incoming priority message from X, message as follows"_
image

image
:cl:
tweak: The notice that you have received a fax is now easier to see as well as headset messages from CC or the Syndicate
/ :cl:

### CODE REVIEW

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -1760,7 +1760,7 @@
 		P.faxmachineid = fax.UID()
 		P.loc = fax.loc // Do not use fax.receivefax(P) here, as it won't preserve the type. Physically teleporting the fax paper is required.
 		if(istype(H) && H.stat == CONSCIOUS && (istype(H.l_ear, /obj/item/radio/headset) || istype(H.r_ear, /obj/item/radio/headset)))
-			to_chat(H, "Your headset pings, notifying you that a reply to your fax has arrived.")
+			to_chat(H, "<span class='warning'>Your headset pings, notifying you that a reply to your fax has arrived.</span>")
```
Comment (Kyep) [inline code/modules/admin/topic.dm]: I'd suggest a class other than 'warning'. 'Warning' messages might be small red font, but they're also some of the most common chat messages. If you want it to stand out, you need to use purple or some other rarely-used color. Also, 'warning' messages are in italics, if memory serves, which makes them harder to read.

Comment (Ty-Omaha) [reply code/modules/admin/topic.dm]: Ok I changed it to the danger class

Comment (Kyep) [reply code/modules/admin/topic.dm]: Include a screenshot of how that looks?

Comment (Ty-Omaha) [reply code/modules/admin/topic.dm]: It's the same class when someone attacks you

Comment (Kyep) [reply code/modules/admin/topic.dm]: I thought that was 'userdanger'.

Comment (Ty-Omaha) [reply code/modules/admin/topic.dm]: Hm let me check

Comment (Ty-Omaha) [reply code/modules/admin/topic.dm]: Yeah I changed it to userdanger

File: code/modules/admin/topic.dm [hunk 2]
```diff
@@ -1760,7 +1760,7 @@
 		P.faxmachineid = fax.UID()
 		P.loc = fax.loc // Do not use fax.receivefax(P) here, as it won't preserve the type. Physically teleporting the fax paper is required.
 		if(istype(H) && H.stat == CONSCIOUS && (istype(H.l_ear, /obj/item/radio/headset) || istype(H.r_ear, /obj/item/radio/headset)))
-			to_chat(H, "Your headset pings, notifying you that a reply to your fax has arrived.")
+			to_chat(H, "<span class = 'specialnotice'>Your headset pings, notifying you that a reply to your fax has arrived.</span>")
```
Comment (Ty-Omaha) [inline code/modules/admin/topic.dm]: this isnt working and I have no idea why

File: code/modules/admin/verbs/randomverbs.dm [hunk 3]
```diff
@@ -181,7 +181,7 @@
 
 	log_admin("[key_name(src)] replied to [key_name(H)]'s [sender] message with the message [input].")
 	message_admins("[key_name_admin(src)] replied to [key_name_admin(H)]'s [sender] message with: \"[input]\"")
-	to_chat(H, "You hear something crackle in your ears for a moment before a voice speaks.  \"Please stand by for a message from [sender == "Syndicate" ? "your benefactor" : "Central Command"].  Message as follows[sender == "Syndicate" ? ", agent." : ":"] <span class='bold'>[input].</span> Message ends.\"")
+	to_chat(H, "<span class = 'specialnotice'>Please stand by for a message from [sender == "Syndicate" ? "your benefactor" : "Central Command"].  Message as follows[sender == "Syndicate" ? ", agent." : ":"] [input]")
```
Comment (moxian) [inline code/modules/admin/verbs/randomverbs.dm]: Unclosed <span>

File: code/modules/admin/topic.dm (hunk 2 above)
Comment (Ty-Omaha) [reply code/modules/admin/topic.dm]: still broken

Comment (Birdtalon) [reply code/modules/admin/topic.dm]: Discord @'d you.

**Pull request merged**